docker-compose.yml

```diff
@@ -4,25 +4,29 @@ services:
       dockerfile: vtn.Dockerfile
       context: .
     ports:
-      - "127.0.0.1:3000:3000"
+      - ${VTN_PORT}:3000
```

Comment:
One more thing that just came to my mind: Is there a reason you want to bind to any interface instead of just loopback? It does not necessarily seem smart to me to expose the development environment to an external interface.
Same for the DB in line 27